user-management.yaml

```
@@ -588,12 +593,71 @@ paths:
           description: |
             Unprocessable Entity  
             types: LoginTokenSignatureInvalid
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+    delete:
+      tags:
+      - "User Management"
+      summary: "Delete a profile picture"
+      description: |
+        Resets a user's profile picture back to the default.
+      operationId: "deleteImage"
+      security:
+        - uc4_token_login: [] 
+        - uc4_token_login_header: []
+      parameters:
+      - name: "username"
+        in: "path"
+        required: true
+        schema:
+          type: "string"
+      responses:
+        "200":
+          description: "OK"
+        "400":
+          description: | 
+            Bad Request  
+            types: MalformedLoginToken, MultipleAuthorizationError, Deserialization, MissingHeader
           content:
             application/json:
               schema:
                 oneOf:
                   - $ref: '#/components/schemas/GenericError'
                   - $ref: '#/components/schemas/DetailedError'
```

```suggestion
            types: MalformedLoginToken, MultipleAuthorizationError
          content:
            application/json:
              schema:
                oneOf:
                  - $ref: '#/components/schemas/GenericError'
```
Deserialization and MissingHeader cannot occur here. Therefore, no DetailedError can be returned.